src/katgpucbf/xbgpu/correlation.py

```diff
@@ -130,7 +130,7 @@ def __init__(
                 f"-DNR_RECEIVERS={self.n_ants}",
                 f"-DNR_RECEIVERS_PER_BLOCK={self._n_ants_per_block}",
                 f"-DNR_BITS={self.input_sample_bits}",
-                f"-DNR_CHANNELS={self.n_channels}",
+                f"-DNR_CHANNELS_PER_SUBSTREAM={self.n_channels_per_substream}",
```

Comment:
I'd prefer not to be messing with the variable names used by TCC - it makes it harder to incorporate updates. Several of the other variables names are also different to what we use, but that's reasonable at interface boundaries with 3rd-party code.

Reply:
This is a good point that I hadn't thought about.